applications/vessel-history/src/features/profile/components/InfoField.tsx

```diff
@@ -35,31 +35,27 @@ interface ListItemProps {
   value?: string
   valuesHistory?: ValueItem[]
   vesselName: string
-  hideTMTDate?: boolean
```

Comment:
why are you deleting all hideTMTDate?

Reply:
Because this story is to display all TMT dates based on the special cases like 19970000 and 19970900

Reply:
Good catch @fertandil87 ! You are right, we should continue hiding the dates on those specific fields. Thanks for catching it!